### PR TITLE
Update examples

### PR DESCRIPTION
**Pull request by witzatom**
Based on helping Katriel with his setup, i did some tweaks and additions to the readme. I put the 'typical' pipeline run up higher as thats the main way to run the pipeline.

Further i completely rewrote the example run script to do a simple run of the pipeline, instead of having a fairly complicated step by step with multiple configurations. Also I use the secrets.cfg to configure the pipeline instead of passing many parameters.

### CODE REVIEW

File: scripts/run-interactive.sh [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+LUIGI="./run_luigi"
+FORETOLD_TOKEN=$(prompt_input "Foretold token")
+SIM_DIR=$(prompt_input "GLEAMviz simulation directory (for example ~/GLEAMviz-data/sims)")
+OUTPUT_DIRECTORY=./data-dir/outputs
+
+cat << EOM > secrets.cfg
+[UpdateForetold]
+foretold_channel = ${FORETOLD_TOKEN}
+
+[ExportSimulationDefinitions]
```
Comment (mbalatsko) [inline scripts/run-interactive.sh:41]: Why this config is saved to secrets.cfg?

**Review by mbalatsko — APPROVED** [review]

File: scripts/run-interactive.sh (hunk 1 above)
Comment (mbalatsko) [reply scripts/run-interactive.sh:41]: Ale asi to neni moc podstatni

Comment (witzatom) [reply scripts/run-interactive.sh:41]: because i dont want to edit the luigi.cfg since that is being commited

Comment (mbalatsko) [reply scripts/run-interactive.sh:41]: Fair point

Comment (mbalatsko) [reply scripts/run-interactive.sh:41]: I think we can merge it

**Comment by witzatom** [comment]
migrated to https://github.com/epidemics/covid/pull/470

**Pull request closed without merging**